.github/workflows/ci.yml

```diff
@@ -77,6 +77,41 @@ jobs:
           cd tmp
           ruby test/run.rb
 
+  host-ibm:
+    if: github.repository == 'ruby/fiddle'
+    name: >-
+      ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-24.04-ppc64le
+          - ubuntu-24.04-s390x
+
+    steps:
+      - uses: actions/checkout@v5
+
+      - name: Set up Ruby
+        run: |
+          sudo apt-get update
+          sudo apt-get install ruby-full bundler libffi-dev
```

Comment:
Can we use `ruby/setup-ruby@v1` on `ubuntu-24.04-{ppc64le,s390x}`?

Reply:
Unfortunately, we cannot use the ruby/setup-ruby for ubuntu-24-04-{ppc64le,s390x} for now, seeing the following logic.

https://github.com/ruby/setup-ruby/blob/3fee6763234110473bd57dd4595c5199fce2c510/common.js#L173-L181

We need to update the ruby/setup-ruby like following case for arm64.

https://github.com/ruby/setup-ruby/commit/fbaef136828f75c08f5f5db5f3f289ee30044a4e

I will work for ruby/setup-ruby to support `ubuntu-24-04-{ppc64le,s390x}`.

Reply:
@kou Do you want to postpone to merge this PR, until the ruby/setup-ruby supports ppc64le/s390x?

Reply:
What are `ubuntu-24-04-{ppc64le,s390x}`, are they self-hosted runners?
I don't see them as GitHub-hosted runners at https://docs.github.com/en/actions/reference/runners/github-hosted-runners at least.
In general, ruby/setup-ruby can only support GitHub-hosted runners.

Reply:
According to https://github.com/IBM/actionspz/blob/main/docs/onboarding.md they are self-hosted.

Also based on https://github.com/IBM/actionspz/issues/4#issuecomment-3076501147 I worry to have issues with limited availability of these runners, and e.g. I wouldn't want them to make the total CI time longer, or have to wait a long time to get such a runner when building some new Ruby version.
Also we already have many platforms and it's causing problems like https://github.com/ruby/setup-ruby/issues/752#issuecomment-3299620427

Nevertheless, feel free to open an issue or discussion at https://github.com/ruby/setup-ruby about this, it's a better place to discuss that.